templates/secrets-key.yaml

```diff
@@ -9,7 +9,7 @@ type: Opaque
 {{- $previous := lookup "v1" "Secret" .Release.Namespace (include "nlweb.secrets.key.name" .) }}
 data:
   nlwSecretKey: {{ .Values.neoload.configuration.secretKey | required "You need to specify a NLW secret key. (--set neoload.configuration.secretKey=mySecretKey)" | toString | b64enc | quote }}
-  {{- if $previous }}
+  {{- if and ($previous) ($previous.data.internalTokenSecret) }}
```

Comment:
Won't you have a null pointer if $previous is defined, but $previous.data is undefined ?

Reply:
You right. I added the check.

Reply:
Good question :)
I will expect that if the Secret exists then the data is prensent because you already have the  `nlwSecretKey`.
I don't see a case where it's not the case but I can miss something.